src/sentry/rules/actions/sentry_apps/notify_event.py

```diff
@@ -62,6 +62,23 @@ def _get_setting_value(self, field_name: str) -> str | None:
             None,
         )
 
+    def _get_sentry_app_installation_uuid(self) -> Any:
```

Comment:
Getting Sentry app installation UUID was moved to a function to avoid redundant code (used in existing `self_validate` and new `render_label` functions